docs/source/tutorials/chat.rst

```diff
@@ -0,0 +1,164 @@
+=================================
+Fine-tuning LLaMA3 with chat data
+=================================
+
+In this tutorial, we'll demystify what prompt templates are and when you'll need them
+and talk about the differences between prompt templating for LLaMA2 and LLaMA3. Then,
+we'll wrap up with a LLaMA3 finetuning example on a custom chat dataset.
+
+.. grid:: 2
+
+    .. grid-item-card:: :octicon:`mortar-board;1em;` You will learn:
+
+      * What is a prompt template and how is it different from special tokens
+      * When to use and not to use a prompt template
+      * How to use your own chat dataset to fine-tune LLaMA3
```

Comment:
Why not just lead with this one, I feel like the other points are too in the weeds and kinda unclear tbh

Reply:
How about:

- Fine tune chat data with llama3
- what prompt templates are and how to use them